index.md

```diff
@@ -16,20 +16,22 @@ title: "Catalog of IDR images formatted as OME-NGFF"
 }
 </script>
 
-<table class="display" id="table">
+<table class="display table" id="table">
     <thead>
 <!-- TODO: should be read from data file -->
         <tr>
             <th>OME-NGFF version</th>
             <th>Thumbnail</th>
             <th>EMBL-EBI S3 key</th>
+            <th>View in IDR</th>
```

Comment:
Since the catalog is primarily about OME-NGFF images, my inclination would be to push provenance information further to the right of the table. Semantically, the closest column would be the `Study` one which links to the top-level `/search` URL.